frontend/packages/akr/src/styles/components/clerkTranslator/_clerk-translator-listing.scss

```diff
@@ -4,4 +4,25 @@
       color: $color-grey-600;
     }
   }
+
+  &__row {
+    td:nth-child(2) {
```

Comment:
Rivi toimii linkkinä vasta nimikentästä oikealle päin katsottuna. Kursori on pointer mikäli ollaan tällä alueella tai checkboxin päällä.